src/modals/cluster-upgrade/ClusterUpgradeDataProvider.js

```diff
@@ -144,16 +148,20 @@ async function upgradeCluster ({
   try {
     await ansiblePlaybookPost(playbookName, ansibleVariables, executionTimeoutInMin)
     getPluginApi().showToast(
-      webadminToastTypes.info,
+      C.webadminToastTypes.info,
       msg.clusterUpgradeOperationStarted({ clusterName })
     )
+    applySearch(C.webadminPlaces.event, C.searchPrefixes.event, [{
```

Comment:
I'm not sure that jumping to the events grid, once the Ansible playbook starts, is the right way to go:

1. Auto jumping from cluster grid to a different dialog without an advanced notice seems like a bad UX and might annoy the user if he planned to continue viewing cluster data or run another cluster upgrade.

2. For checking upgrade status - we do plan to add a graphic progress bar on clusters grid for displaying the upgrade process.

Maybe instead of jumping to events tab, just add a link for events tab with filtering by the correlation id.

Reply:
Jumping to the events grid has been removed. Any further status display or navigation will be handled in a separate PR.